app/models/manageiq/providers/google/cloud_manager.rb

```diff
@@ -18,7 +20,15 @@ def self.region_required?
   end
 
   def supported_auth_types
-    %w(oauth)
+    %w(
+      oauth
+      service_account
```

Comment:
I don't think this will work if the column and lookup method are referencing `service_account_json` while this is called `service_account`.

I actually like `service_account` better than `service_account_json`.

Can you rename the field name and the accessor method to `service_account` and `authentication_service_account`, respectively?